### ISSUE TITLE
[Infrastructure] Remove or improve Play Again button for Spectators

### ISSUE THREAD

**Issue by gk-patel** [freeboardgames/FreeBoardGames.org#726]
Either remove or improve the Play Again button for Spectators, 
IMPROVE: allow them to wait for the game to start 
REMOVE: dont show the play again button if the user is a spectator. 

I thinking REMOVE would be simpler, but currently if the "play again" button is clicked, then, 
![image](https://user-images.githubusercontent.com/42677721/102881293-ef6c7d80-444c-11eb-93df-c8bc3a4ca2dc.png)
Then this image is shown

**Comment by flamecoals** [comment]
I think that if it was in production, the button would just do nothing (and throw an error on the terminal). We shouldn't be showing it at all I think.

**Comment by gk-patel** [comment]
So you can remove it! 

BTW, how does the Play Again button work, does it generate the new game key based on the previous one ?

**Comment by flamecoals** [comment]
Yes, there is an endpoint that given a room id generates a new one or get the last generated one. 
I can try to fix this later, but I am prioritizing customization for now, spectators is not a very used feature... Feel free to take a stab at it and send me questions if you have any.
